op-batcher/batcher/driver.go

```diff
@@ -264,49 +262,54 @@ func (l *BatchSubmitter) calculateL2BlockRangeToStore(ctx context.Context) (eth.
 func (l *BatchSubmitter) loop() {
 	defer l.wg.Done()
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
I think we can just replace this with
```suggestion
	ctx := context.TODO()
```
(or `context.Background()`). `TODO` because we might want to add a second force-shutdown context to the batch submitter's interal state. The current `l.ctx` now acts like a orderly-shutdown context because it now only cancels block loading.